api/bin/chainflip-cli/src/settings.rs

```diff
@@ -235,6 +235,15 @@ pub enum CliCommand {
 		#[clap(help = "The epoch to check, default to the current one")]
 		epoch_index: Option<EpochIndex>,
 	},
+	#[clap(about = "Request an internal transfer of FLIP to another account")]
+	RequestInternalTransfer {
+		#[clap(help = "The amount to transfer")]
+		amount: Option<f64>,
+		#[clap(help = "The Ethereum address under which condition we transfer the FLIP")]
+		destination: String,
+		#[clap(help = "The account ID to transfer the FLIP to")]
+		account_id: String,
```

Comment:
Similar comment to above: naming could be better / more descriptive. For example, `destination` is a confusing name for a thing that is not the destination...

Would it make sense to extend the existing `redeem` method instead of adding an entirely new one? 

`chainflip-cli redeem --exact=1000 $ETH_ADDRESS --internal-destination=$ACCOUNT_ID`